src/util.ts

```diff
@@ -109,6 +109,9 @@ export function spawn(command: string, args?: Array<string> | null, options?: Sp
 }
 
 export async function getElectronVersion(packageData: any, packageJsonPath: string): Promise<string> {
+  if (packageData.config['electron-version']) {
+    return packageData.config['electron-version']
```

Comment:
I believe there should be some kind of validity checking and error reporting of the user defined version.